src/org/labkey/test/pages/assay/ChooseAssayTypePage.java

```diff
@@ -62,8 +62,8 @@ public ReactAssayDesignerPage selectAssayType(String name)
         waitForElementToBeVisible(elementCache().specialtySelectLocator);
         selectOptionByText(elementCache().specialtySelect, name);
 
-        waitFor(()-> elementCache().selectButton.getText().toLowerCase().contains(name),
-            "took too long to update", 2000);
+        waitFor(()-> elementCache().selectButton.getText().toLowerCase().contains(name.toLowerCase()),
+            "took too long for the select button text to contain the assay name", 2000);
```

Comment:
Why not include the name in the error message?
```
waitFor(()-> elementCache().selectButton.getText().toLowerCase().contains(name.toLowerCase()),
            String.format("Took too long for the select button text to contain the assay name '%s'.", name), 2000);
```

Reply:
That's a good idea, I'll do it